core/src/main/scripts/importer/cbioportal_common.py

```diff
@@ -554,8 +554,22 @@ def flush(self):
                                  record.getMessage())
             if identifying_tuple not in grouping_dict:
                 grouping_dict[identifying_tuple] = []
-            grouping_dict[identifying_tuple].append(record)
 
+            #Add a tuple that keeps the relation between the line number,
```

Comment:
The first thing that distracted me was the different comment style from the ones above and below, but I should probably try not to let that distract me.